snapcraft.yaml

```
@@ -80,13 +109,9 @@ parts:
       - findutils
       - liblz4-tool
       - libc-dev
-      - on amd64 to arm64:
-        - gcc-aarch64-linux-gnu:amd64
-        - libc6-dev-arm64-cross:amd64
-      - on amd64 to armhf:
-        - gcc-arm-linux-gnueabihf:amd64
-        - libc6-dev-armhf-cross:amd64
-
+      - try:
+          - gcc-aarch64-linux-gnu:amd64
+          - libc6-dev-arm64-cross:amd64
```

Is this branch targetting only arm64, or only armhf, or both?

Cause above triplet is set to armhf, yet only arm64 cross-compiler is installed. Unless things that are build here need compilation for armhf tripplet irrespective of armhf/arm64 (freestanding?! but then it would not run on a subset of AArch64 cpus without the optional AArch32 instruction set)

---

ah thanks for catching this, this is actually a problem, since this PR is targeting the armhf branch, this should be

```suggestion
          - libc6-dev-armhf-cross:amd64
```
(or whatever the actual packages are called)

but to answer your question, the `try` is always executed on all architectures, but it's not treated as fatal if snapcraft doesn't find this package available when it is building. This is because the `on ... to ...` support in snapcraft is broken because --target-arch is what sets the arch in `to ...` but we can't set that anymore except when building dangerously.

---

fixed this